### PR TITLE
Add SNMP OIDs

### PR DESCRIPTION
**Pull request by Axenow**
Add SNMP for: Cisco Catalyst WS-C3850-48T, WS-C3850-48P, Cisco AIR-3702E
Add ios_codes and nxos_codes in doSwitchSNMPmining.

Some devices and os versions should be added to dictionary.

### CODE REVIEW

File: wwwroot/inc/snmp.php [hunk 1]
```diff
@@ -2452,6 +2452,12 @@
 		'text' => 'WS-C4948E: 48 RJ-45/10-100-1000T(X) + 4 SFP+/10000 + 1 RJ-45/100TX (OOB mgmt)',
 		'processors' => array ('catalyst-chassis-uplinks-10000SFP+', 'catalyst-chassis-uplinks-1000T', 'catalyst-chassis-mgmt'),
 	),
+    '9.1.1643' => array
+    (
+        'dict_key' => 2190,
+        'text' => 'WS-C3850-48T: 48 RJ-45/10-100-1000T(X) + 4 SFP+/10000 or 4 SFP/1000',
+        'processors' => array ('catalyst-stack-uplinks-10000SFP+', 'catalyst-chassis-mgmt', 'catalyst-stack-any-1000T'),
+    ),
```
Comment (adoom42) [inline wwwroot/inc/snmp.php]: use tabs instead of spaces

File: wwwroot/inc/snmp.php [hunk 2]
```diff
@@ -3918,6 +3936,8 @@ function doSwitchSNMPmining ($objectInfo, $device)
 			'12.2' => 252,
 			'15.0' => 1901,
 			'15.1' => 2082,
+			'15.2' => 50034,  // Should be created in dictionary
+			'15.3' => 50039,  // Should be created in dictionary
```
Comment (adoom42) [inline wwwroot/inc/snmp.php]: you should add it to the dictionary

File: wwwroot/inc/snmp.php [hunk 3]
```diff
@@ -2486,6 +2492,18 @@
...
 	),
+	'9.1.1745' => array
+	(
+		'dict_key' => 50038,  // Should be created in dictionary
+		'text' => 'WS-C3850-48P: 48 RJ-45/10-100-1000T(X) + 4 SFP+/10000',
+		'processors' => array ('catalyst-stack-uplinks-10000SFP+', 'catalyst-chassis-mgmt', 'catalyst-stack-any-1000T'),
+	),
+	'9.1.1873' => array
+	(
+		'dict_key' => 50051,  // Should be created in dictionary
```
Comment (adoom42) [inline wwwroot/inc/snmp.php]: you should add it to the dictionary

**Comment by Axenow** [comment]
I fixed space and change it to tabs

**Comment by Axenow** [comment]
Add one more switch: Cisco Nexus 5548UP

**Comment by infrastation** [comment]
This pull request cannot be reviewed/merged without cleanup and a rebase on the current branch first. Is anybody willing to do this?

**Comment by infrastation** [comment]
Please reopen when you can work on finishing this.

**Pull request closed without merging**